api/src/main/java/com/velocitypowered/api/event/Subscribe.java

```diff
@@ -11,4 +11,19 @@
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.METHOD)
 public @interface Subscribe {
+
+    /**
+     * The order events will be posted to this listener.
+     *
+     * @return the order
+     */
+    PostOrder order() default PostOrder.NORMAL;
+
+    /**
+     * If cancelled events shouldn't be posted to this listener.
+     *
+     * @return if cancelled events should be ignored.
+     */
+    boolean ignoreCancelled() default false;
```

Comment:
Velocity's does not have a concept of a "cancelled event". It does, however, have the closely related "resulted event".